containers/podman-desktop
# chore: graduate onboarding from experimental

chore: graduate onboarding from experimental

### What does this PR do?

Removes the experimental flag and "graduates" onboarding out of
experimental.

Onboarding will now show without having to add the experimental flag to
your settings.

### Screenshot/screencast of this PR




https://github.com/containers/podman-desktop/assets/6422176/bb447810-0309-4309-87c8-a3251716b128



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/4012

### How to test this PR?



1. Uninstall podman
2. See that we now have the "setup" button for podman / onboarding.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
